include/openthread.h

```diff
@@ -157,83 +163,123 @@ const char *otGetVersionString(void);
  * This function initializes the OpenThread library.
  *
  * This function initializes OpenThread and prepares it for subsequent OpenThread API calls.  This function must be
- * called before any other calls to OpenThread.
+ * called before any other calls to OpenThread. By default, OpenThread is initialized in the 'enabled' state.
+ *
+ * @param[in]    aContextBuffer      The buffer for OpenThread to use for allocating the otContext structure.
+ * @param[inout] aContextBufferSize  On input, the size of aContextBuffer. On output, if not enough space for otContext,
+                                     the number of bytes required for otContext.
+ *
+ * @retval otContext*  The new OpenThread context structure.
+ *
+ */
+otContext *otInit(void *aContextBuffer, uint64_t *aContextBufferSize);
+
+/**
+ * This function disables the OpenThread library.
+ *
+ * Call this function when OpenThread is no longer in use.
+ *
+ * @param[in] aContext  The OpenThread context structure.
+ *
+ */
+void otFreeContext(otContext *aContext);
```

Comment:
`otEnable()` and `otDisable()` were intended to serve the purpose of `otInit` and `otFreeContext` proposed here.  I see you have repurposed `otEnable()` to call `otInterfaceUp()` and `otThreadStart()`.  I'm not sure we need to provide a convenience function that collapses the two calls.